### ISSUE TITLE
RMSE keeps increasing after 25 epochs training while disparity prediction looks fine

### ISSUE THREAD

**Issue by WilliamKRobert** [ClubAI/MonoDepth-PyTorch#23]
I am training your code on the KITTI eigen split with the default hyper parameters in the README. At the beginning, everything was fine and the loss decreased to 1.5, rmse was around 10 meters. However, after 25 epochs training (learning rate is kept as 1e-4), continuous increase of rmse was observed, like 200~800 meters. I checked the predicted disparity map and it looked fine. A further check against the ground truth indicated that the crazy rmse error was due to small disparities in some regions, especially textureless areas. 

I thought disparity smoothness loss should prevent jump of disparity prediction, right? Any ideas?
